bluecellulab/analysis/plotting.py

```diff
@@ -2,6 +2,140 @@
 
 import matplotlib.pyplot as plt
 import pathlib
+import numpy as np
+from typing import Optional, Tuple
+from bluecellulab.tools import calculate_rheobase
+from bluecellulab.circuit.circuit_access import EmodelProperties
+from bluecellulab.cell import Cell
+from bluecellulab.simulation.neuron_globals import NeuronGlobals
+from bluecellulab import Simulation
+
+
+def generate_cell_thumbnail(
+    template_path: str,
```

Comment:
Could you reduce the number of function arguments by passing template_params instead?
See https://github.com/openbraininstitute/BlueCelluLab/blob/main/bluecellulab/cell/template.py#L48